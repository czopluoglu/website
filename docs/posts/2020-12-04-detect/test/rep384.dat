10100001101101111011
01110001111111110111
11100000001010001110
11101100111110001110
11010111110111111111
11011101111111111110
11111011011101110111
01001001111011011011
10100111011001110011
10011111111010001111
11000011001100000111
01100000001001000100
10000011001111111010
10100101110010001111
11101111110100011111
01001001001100111111
11100111001110101111
11011100111011101111
00001001101110110110
11001000010100010101
01101100011101000111
01111101111000101010
00000001000010000110
10111001100110000010
10011000000010001000
11111111010100011111
10011101110000110101
10010001111101100110
01111001101000000110
01111111011110110111
00101000001010100000
10111111110111101111
01111000111111111110
11111101111100100010
01101000101100001011
11100000101000110010
11000101001111111110
11110101111011110110
10011000000010000011
10000110001010111110
01111001011010000001
01011100001110001111
11111001001010000110
11110111011101101101
11000111101001000001
00100001001001000000
10010000001111010011
11000011101111111011
01111001110010001110
01101101111110101101
11111100111011010010
01101010011010100011
01101011111000100111
10111100011110110111
01101001101100101111
00100100011001101000
10001111111111110111
11100001111000001110
10000001001111010101
00110000000110000010
00000011001000110110
01110110111000100110
11110111101001101010
10010101110100001010
10111101111101111111
11010101100010010111
11001111001111111111
10000100111010010111
11000010001001110110
11110001111110111100
01001111101110101111
00010111101000110000
00111000111111101011
11011100100000110010
00100111101111111111
11111111001110011111
10001100101101001110
01000000011110010111
10110000011110111011
01100011101111111101
10111101110000000010
00111110101010010110
11000101111000111101
10111111111111110111
00110011001111100101
01010111001111001111
00100001011111110111
10011101110101100111
11010000101101111101
11000001001111100011
10111011010011101100
11011111010000110011
00000000001110000100
11011010010000011100
00010001111011100110
10000100010010000011
11111101010100000101
11101111110111011011
11000001000111001111
10110000000100101011
10000001010010001111
10111101101001110100
10100101111110011111
01111001101111111111
10100001110101000100
01101010011110011110
01100001111110000010
10100110101010001101
10111011100100110001
11111001001110111110
00001011111010000010
10000000001000100010
00110001111110000110
10010111101011100011
00001110000110110011
10111100101110010001
01110001111101000111
01011000011110111000
11101111111111111111
01110000111000100110
00000000001100011110
11001001001001100101
00010000001110111110
10101101001111010110
11011000011111010111
00111100001100011111
11001101000010100011
01010111111110101010
11101011001111100110
00001101101111110010
10011101111010000010
00100000101010100111
11000001111010010111
10111101011111100101
01001001011110101101
11111011101100100011
11001101011111110000
10000110010110100011
11010111110011110100
10111111010001010111
00000010101000000010
00101100111111000110
11111111011001000000
11011010011100101111
00111010000111011110
10001000110110000010
10001110111010001011
01101111000010110111
01010001001110111110
10001100111000000010
00101011111100001001
11001101110111101111
01011111111010111110
01100111001110111101
00001010000110100110
11000101000000000010
01110100011010000000
10000100000001101100
11111101101101100110
10100001001000100001
01110011010101001110
11111001111111111110
10000001011111101011
01011001010101101111
11011001100000110011
11000001011000111111
10101000110110111100
10000000011000001111
10001000001111100111
00011100101010100111
10000000011110100110
11111111101111000111
11011100001011000000
11100000111011000111
01001001111101001010
10110001011111101100
10110001001001111111
00000001001000000011
00011010010110110110
00010000011110000001
10011001111010001010
00000111000111000111
10000111101000011111
00011000000010101111
01110101111111011111
01101100101100101110
11110110011100101001
10001000000011111111
00011000000000101011
10111000001010001111
11010000011001100101
11100001001111001111
10101001110011101111
10010010000000000110
11000011000100100110
10001000011100001110
10111011100000111010
01000001000010010010
11000000101100000011
10001100111110110000
10100101111110100000
10111111111011100110
11011011100000110001
00000000101010110110
00100110001101110111
11110111111110111010
10100111011110110011
00000000000010001001
10000101111101110100
00010101111000100011
10000001111111101110
10111010110111111010
00110010000010101110
01110001001011111111
11001011001111110100
00111000101100101110
10000000001110100101
00110101011101110110
00110000001000001000
10001000011101110011
01010011111001000011
00111100011111100111
11001010111010100110
01101010011011111111
01101101111100010111
01000000111110111011
01111101101111101101
10001000011111101111
11011001101111101111
11100100111010000010
11010011111101111110
01000011001100111010
10000111011101110011
01100100001010101110
01111100001010010010
10110011011111011011
11000001000100001110
01001101101101101111
01011101111111111101
11110100111110010011
10011101000110001111
10100001001100110110
10001000001010101110
01000100110110011101
00000010000000001101
00000001011100101110
01001000000000000001
01011100000000000000
10001101001001000110
10000010000001001111
