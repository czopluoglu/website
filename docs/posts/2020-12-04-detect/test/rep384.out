-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep384.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.32
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8222
                   IDN index value:   0.7579
                           Ratio r:   0.6623

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    3    3    3    1    1    3    3
    2    4    4    1    1    1    4    4    4    4

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    7    8   14   15   16

   -----------CLUSTER  2-------------

    2   11

   -----------CLUSTER  3-------------

    3    4    5    6    9   10

   -----------CLUSTER  4-------------

   12   13   17   18   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++-+-----+-+-----+
   +d+-++-----+------++
   ++d++-+----++-+----+
   +-+d+--+------+--+-+
   -+++d+-+-+----+-++--
   ++--+d++------++----
   --+--+d+--+-+---++-+
   ---++++d+---++++----
   -------+d+++++------
   ----+---+d++++------
   ------+-++d+++-+--+-
   +++-----+++d++-----+
   --+---++++++d+----+-
   +------++++++d-+----
   --++++-+------d+++-+
   -----+-+--+--++d+++-
   ----+-+-------++d+++
   ---++-+-------+++d++
   -+--------+-+--+++d-
   ++++--+----+--+-++-d
 
 No cross validation for this DETECT run
